setup.cfg

```diff
@@ -13,7 +13,6 @@ classifiers=
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Operating System :: OS Independent
```

Comment:
Dang just realized this now: we should set `python_requires` here.

Reply:
Yep, totally missed it too. I'll submit a second PR later today to add it and will take the chance to also fix some minor typos and little things of the sort.

Reply:
Cool, thanks!

Reply:
Done (#41). On a side note, I wonder if we are ready for 0.15.0 or it would be better to wait a little more.

Reply:
Thanks!

I'm OK with a making a new release, up to you. 👍